services/director-v2/src/simcore_service_director_v2/models/schemas/constants.py

```diff
@@ -5,7 +5,6 @@
 DYNAMIC_SIDECAR_SERVICE_PREFIX: Final[str] = "dy-sidecar"
 DYNAMIC_PROXY_SERVICE_PREFIX: Final[str] = "dy-proxy"
 DYNAMIC_VOLUME_REMOVER_PREFIX: Final[str] = "dy-volrm"
```

Comment:
since you created that file with a constant in service-library. I think it would make sense to move also these other prefixes no?
And again I think it should not be called ```docker_constants.py``` but more something like ```dynamic-sidecar-constants.py```

Reply:
I think they can be moved to a new volume, but as said before I think it reads better like this when importing.

Reply:
@pcrespov any help here?

Reply:
Any module with only constants (i.e. that do NOT introduce any dependencies) would be fine. Your call ...